kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -16,7 +16,7 @@ pushd $appliance_root
 popd
 
 mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $vmdb_root
+git clone --separate-git-dir /tmp/manageiq.git <%= @manageiq_checkout.remote %> $vmdb_root
```

Comment:
@geertj If that's the case, can you please also do that with the other git repos that we clone?  (manageiq-appliance and manageiq-ui-self_service)